TDA-holding-message (#3866)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

[Find out more about the qualifications you need to be a teacher in England](/is-teaching-right-for-me/qualifications-you-need-to-teach).

<div class="inset">

<p>If you're a non-UK citizen who wants to study in England, you can visit the <a href="https://www.ukcisa.org.uk/">UK Council for International Student Affairs</a> for information about studying at an English university.</p>

> </div>

Reviewer: The Find component on this page links to UCAS for undergrad degrees - given that we can't move it away from the bottom of the page for now, I would suggest re-wording it slightly so it is clear that it relates to undergraduate degrees and not TDAs?

Reviewer: Only on this page :-)

> ## Teacher Degree Apprenticeships (TDAs)

Reviewer: ```suggestion
## Teacher degree apprenticeships (TDAs)
```

> If you do not have a degree and want to become a teacher you may be eligible to apply for a TDA. This new course allows you work in a school and earn a salary while getting a batchelor's degree and QTS. Applications for both primary and secondary are expected to launch in autumn 2024 and training is due to start in autumn 2025. 

Reviewer: ```suggestion
If you do not have a degree and want to become a teacher you may be eligible to apply for a TDA. This new course allows you work in a school and earn a salary while getting a bachelor's degree and QTS. Applications for both primary and secondary are expected to launch in autumn 2024, with training due to start in autumn 2025. 
```

Reviewer: Worth splitting `Applications for....` onto a new para?

Reviewer: And do we need `and want to become a teacher`?

Author: @gemmadallmandfe I didn't know if it might be good to include it for SEO but I can take it out!

Reviewer: we might be better with `become a teacher` traffic going to Steps though?

I think `**teacher** degree **apprenticeships**` will be strong enough re SEO

> <a href="https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/">Find out more about Teacher Degree Apprenticeships(TDAs)</a>

Reviewer: ```suggestion
<p><a href="https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/">Find out more about Teacher Degree Apprenticeships(TDAs)</a>.</p>
```
GDS style to include full-stop outside link text
